espresso-server/build.gradle

```diff
@@ -1,7 +1,25 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.31'
+    ext.getIntegerProperty = {String propertyName, int defaultValue ->
+        Object propertyValue = project.getProperties().get(propertyName)
+        if (propertyValue == null) {
+            return defaultValue
+        } else {
```

Comment:
how about using the [elvis](https://mrhaki.blogspot.com/2009/08/groovy-goodness-elvis-operator.html) operator instead?

Reply:
I don't think using elvis operator is good idea here, so I replaced this block with ternary operator.